saltlint/rules/NoNbspRule.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+
+from saltlint.linter import SaltLintRule
+
+
+class NoNbspRule(SaltLintRule):
+    id = '212'
+    shortdesc = 'Most files should not contain non-breakable spaces'
+    description = 'Non breakable spaces can cause unexpected display issues, use spaces'
+    severity = 'LOW'
+    tags = ['formatting']
+    version_added = 'v0.0.11'
```

Comment:
The next version might as well be `v0.1.0` or `v1.0.0` so I would suggest using `develop`, whenever we release a version we will replace all `develop` entries with the correct version.

```suggestion
    version_added = 'develop'
```

Reply:
👌